dask/dataframe/groupby.py

```diff
@@ -413,6 +421,17 @@ def __getattr__(self, key):
         except KeyError as e:
             raise AttributeError(e)
 
+    def fillna(self, value):
+        meta = self.obj.head(1).groupby(self.index).fillna(value)
+        if isinstance(meta, pd.DataFrame) and self._slice is not None:
+            meta = meta[self._slice]
+        columns = meta.columns if isinstance(meta, pd.DataFrame) else meta.name
+
+        result = map_partitions(_fillna, self.obj, value, self.index, columns,
+                                meta=meta, token='fillna')
```

Comment:
I think it won't be sufficient for any fill method, others will also fail in certain cases.
I think the method parameter is not supported in da.DataFrame``,  I kept in unsupported here as well.

 I can try making this PR a full "support fillna methods" in both.